include/openssl/cms.h

```diff
@@ -491,14 +491,16 @@ int ERR_load_CMS_strings(void);
 # define CMS_R_TYPE_NOT_ENVELOPED_DATA                    146
 # define CMS_R_UNABLE_TO_FINALIZE_CONTEXT                 147
 # define CMS_R_UNKNOWN_CIPHER                             148
-# define CMS_R_UNKNOWN_DIGEST_ALGORIHM                    149
+# define CMS_R_UNKNOWN_DIGEST_ALGORITHM                   149
+# define CMS_R_UNKNOWN_DIGEST_ALGORITM /*compat*/ CMS_R_UNKNOWN_DIGEST_ALGORITHM
```

Comment:
Any compat things must go *before* the BEGIN ERROR CODES comment.  Otherwise they will be wiped out.